### ISSUE TITLE
Don't ignore kubeadm init failures on the master node

### ISSUE THREAD

**Issue by Tomsons** [geerlingguy/ansible-role-kubernetes#28]
Sometimes, kubeadm init fails for some reasons, for example swap is on, preflight errors, image pull timeout due to corporate proxy configuration that has not been ported to docker etc ..., don't ignore it using the `failed_when: False` flag, it's very confusing to see the playbook run successfully for the init command and crash later on when it comes to the task `Symlink the kubectl admin.conf to ~/.kube/conf.` without and explicit stack trace of what went wrong and why

https://github.com/geerlingguy/ansible-role-kubernetes/blob/5c4eea46ed3cc7fc9a2e3ddb2fef11e1b2fdb773/tasks/master-setup.yml#L12

**Comment by rdxmb** [comment]
The problem here could be `--ignore-preflight-errors={{ kubernetes_ignore_preflight_errors }}` , where `{ kubernetes_ignore_preflight_errors }}` is `all` by default at the moment.

**Comment by rdxmb** [comment]
Also, removing the lines 

```
  register: kubeadmin_init
  failed_when: false
  when: not kubernetes_init_stat.stat.exists
```

would end in "real" errors, e.g. when having vars configured uncorrectly. What's the intention for those lines?

**Comment by rdxmb** [comment]
There is another thing in this task: Initializing the cluster will skip the output-task - even if there are errors:

```
TASK [kubernetes : Initialize Kubernetes master with kubeadm init.] *************************
changed: [eins.mbr.intern]

TASK [kubernetes : Print the init output to screen.] ****************************************
skipping: [eins.mbr.intern]

TASK [kubernetes : Ensure .kube directory exists.] ******************************************
ok: [eins.mbr.intern]

TASK [kubernetes : Symlink the kubectl admin.conf to ~/.kube/conf.] *************************
fatal: [eins.mbr.intern]: FAILED! => {"changed": false, "msg": "src file does not exist, use \"force=yes\" if you really want to create the link: /etc/kubernetes/admin.conf", "path": "/root/.kube/config", "src": "/etc/kubernetes/admin.conf", "state": "absent"}
```

**Comment by geerlingguy** [comment]
This has been fixed in #36.
